src/ndb_p1_section.c

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Roman Gezikov <rgezikov@exafore.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+
+#include "ndb_p1_section.h"
+
+#include <string.h>
+#include "ndb.h"
+
+#define GPS_L1_EPHE_FILE_NAME "gps_l1_ephe"
```

Comment:
Another design question: Would it be better / easier to NOT split out data elements that are common across all codes? E.g. why not just do ephemeris_t ephemeris[PLATFORM_SIGNAL_COUNT] and then have a generic ephemeris_get(gnss_signal_t sid, ephemeris_t *ephemeris) function?